packages/modals/src/styled/StyledFooterItem.ts

```diff
@@ -7,15 +7,25 @@
 
 import styled from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import ModalStyles from '@zendeskgarden/css-modals';
 
 const COMPONENT_ID = 'modals.footer_item';
 
 export const StyledFooterItem = styled.span.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: ModalStyles['c-dialog__footer__item']
+  'data-garden-version': PACKAGE_VERSION
 })`
+  display: flex;
+  margin-left: ${props => props.theme.space.base * 5}px;
+  min-width: 0;
+
+  &:first-child {
+    margin: ${props => !props.theme.rtl && 0};
+  }
+
+  &:last-child {
+    margin: ${props => props.theme.rtl && 0};
+  }
```

Comment:
For these values you could dynamically compute the correct child. Like this

```
&:${props => props.theme.rtl ? 'last' : 'first'}-child {
  ...
}
```

Reply:
Thanks for the suggestion. I tried it, but the linter is giving me some problems 🤔.

<img width="473" alt="Screen Shot 2020-01-28 at 4 25 30 PM" src="https://user-images.githubusercontent.com/1811365/73317417-d29eda00-41ea-11ea-8ffd-49893231708f.png">

Reply:
Tried various forms of `prettier-ignore` and `stylelint-disable`, but the error still shows when running `yarn lint`. There could be an [issue](https://github.com/stylelint/stylelint/issues/4401) with stylelint.

Reply:
On second thought, I have removed these styles in favor of the one that JZ suggested in another thread.

```css
margin-${props => (props.theme.rtl ? 'right' : 'left')}: ${props => props.theme.space.base * 5}px;
```